pyproject.toml

```
@@ -11,6 +11,7 @@ repository = "https://github.com/open-resources/problem_bank_helpers"
 include = [{ path = "data/", format = ["sdist", "wheel"]}]
 
 [tool.poetry.dependencies]
+
 python = "^3.9"
```

```suggestion
python = "^3.10"
```
Prairielearn runs on 3.10, so we should actually target that version of python to the best of our abilities (as much as its a moving target) - this one may require the lockfile being regenerated after to satisfy poetry, not sure